taskvine/src/bindings/python3/ndcctools/taskvine/task.py

```diff
@@ -992,13 +992,30 @@ class FunctionCall(Task):
     def __init__(self, library_name, fn, *args, **kwargs):
         Task.__init__(self, fn)
         self._event = {}
-        self._event["fn_kwargs"] = kwargs
         self._event["fn_args"] = args
+        self._event["fn_kwargs"] = kwargs
         self.set_time_max(900)     # maximum run time for function calls is 900s by default.
```

Comment:
@JinZhou5042 @BarrySlyDelgado Is there a reason this is set to 900? I feel this will trip users as it is a hidden parameter. Could we leave the max time unspecified?

Reply:
I'm not sure if there was a reason. I think it was something that was carried from previous iterations.

Reply:
Sounds good, I'll drop it then!